localization/rssguard_de.ts

```diff
@@ -279,9 +281,12 @@
     <message numerus="yes">
         <source>Click me to add feeds from this website.
 This website contains %n feed(s).</source>
-        <translation><numerusform>Zum Hinzufügen der Feeds auf dieser Webseite hier klicken.
-Diese Webseite enhält %n Feed.</numerusform><numerusform>Zum Hinzufügen der Feeds auf dieser Webseite hier klicken.
-Diese Webseite enhält %n Feeds.</numerusform></translation>
+        <translation>
```

Comment:
Please, translations should be added only via Transifex, because when I run localizations-merging script it automatically downloads all translations from transifex and overwrites local .ts files (except english which is "source" language).

After I merge your PR (which looks great!), I will push new english strings to transifex and you can then translate them [here](https://www.transifex.com/martinrotter/rssguard/language/de/).